src/components/applications/ApplicationsView.js

```diff
@@ -36,7 +38,7 @@ export const ApplicationsView = ({getApplications, isHR}) => {
                     view={
                         <ColumnAndDetailsLayout
                             details={selectedApplication ?
-                                <ApplicationDetails application={selectedApplication} isHR={isHR} /> :
+                                <ApplicationDetails application={selectedApplication} isHR={isHR} reload={() => setReload(!reload)} /> :
```

Comment:
Tak myślę, że fajny utilowo byłby jakiś hook useBoolean, coś w stylu
```
const useBoolean = (initialValue) => {
  const [value, setValue] = useState(initailValue)
  
  const toggle = () => setValue(!value)
  const setFalse = () => setValue(false)
  const setTrue = () => setValue(true)

  return [value, toggle, setTrue, setFalse]
}
```

I wtedy w takich sytuacjach wystarczy sobie wziąć tego toggla, mniej kodu i imo fajne

Reply:
Bardziej bym zrobił useReload, bo jednak useBoolean to trochę za ogólne

Reply:
Ogólnie zrobiłem, ale nie zadziałało. Na razie nie bawiłbym się w to, ewentualnie zostawił jako improvement